recipe/build_pytorch.sh

```diff
@@ -69,6 +76,19 @@ export USE_SYSTEM_SLEEF=1
 export BUILD_CUSTOM_PROTOBUF=OFF
 rm -rf $PREFIX/bin/protoc
 
+if [[ "${target_platform}" != "${build_platform}" ]]; then
+    rm -rf ${PREFIX}/bin/protoc-*
+
+    cp ${BUILD_PREFIX}/bin/protoc ${PREFIX}/bin/protoc
+    # libprotobuf 4.23.3 ships out the file protoc-23.3.0
+    # for example and refers to it explicitely
+    # not very portable, but i want to get through the build process
+    cp ${BUILD_PREFIX}/bin/protoc-23.3.0 ${PREFIX}/bin/protoc-23.3.0
+    # if [[ -x "${CONDA_PREFIX}/bin/protoc-*" ]]; then
+    #     cp ${BUILD_PREFIX}/bin/protoc-* ${PREFIX}/bin/
+    # fi
```

Comment:
```suggestion
    # point to a usable protoc if we're cross-compiling
    export Protobuf_PROTOC_EXECUTABLE="$BUILD_PREFIX/bin/protoc"
```

Something like this should work in principle (it's the name of a variable that CMake should respect, but AFAICT we're calling cmake through pip)

Reply:
The thing is, i've tried to specify this, in five different spellings, above...

Reply:
Patch it into the cmake invocation that's behind setup.py?

Reply:
the 30 min iteration cycle I have makes it essentially impossible to test this for me. 

The old mac I have just isn't always plugged in.